src/components/Filter/FilterRow.tsx

```diff
@@ -19,10 +19,16 @@ interface FilterRowInterface extends FilterListInterface {
   onDelete: (index: number) => void;
 }
 
+const LOGIC = [
+  { key: 'and', label: 'And' },
+  { key: 'or', label: 'Or' },
+];
+
```

Comment:
enum will fit better here. at the compile time it's converted into JS object, so you can use it in runtime

```suggestion
enum Logic = {
  and = "And",
  or = "Or",
};

```